src/geckomat/gather_kcats/getStandardKcat.m

```diff
@@ -150,14 +150,13 @@
 
 % Only add if not geckoLight & getStandardKcat was not run earlier
 if ~any(strcmp(model.mets,'prot_standard'))
-    % Add a new gene to be consistent with ec field named standard
-    proteinStdGenes.genes = 'standard';
-    if isfield(model,'geneShortNames')
-        proteinStdGenes.geneShortNames = 'std';
-    end
-    model = addGenesRaven(model, proteinStdGenes);
-
     if ~model.ec.geckoLight
+        % Add a new gene to be consistent with ec field named standard
```

Comment:
Removing this will induce a difference in data between `model.ec.genes` and `model.genes`

Reply:
You're right, might be best to revert this change. It caused problems with I/O a model (as the gene will not be in any grRule (and it shouldn't!)), but this will actually be solved here: https://github.com/SysBioChalmers/RAVEN/pull/494.